src/custom/pages/Faq/index.tsx

```diff
@@ -1,6 +1,8 @@
-import React from 'react'
+import React, { useCallback, useRef, useState } from 'react'
 import Page, { Content, Title } from 'components/Page'
 import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { HashLink } from 'react-router-hash-link'
```

Comment:
we have this internally already, why add a library?

Reply:
As we discussed, this is a pretty simple library and straight fw, over 100K weekly downloads so looks like good tested. This adds on top smooth scroll.

Im working on a PR that will do a refactor, of the two links. Let me do it as an independent PR.

Reply:
https://github.com/gnosis/gp-swap-ui/pull/507